llvm/test/CodeGen/X86/windows-seh-EHa-PreserveCFG.ll

```diff
@@ -0,0 +1,81 @@
+; XFAIL: *
```

Comment:
Hi @HaohaiWen , 

This test passes on our stage2 builders ([example](https://luci-milo.appspot.com/ui/p/fuchsia/builders/prod/clang-linux-arm64/b8759359812022093553/overview)). 

Can you revert and recheck that this test performs as expected on 2stage builds?

Reply:
I'll reland it with fix.